src/components/Dashboard/AddressListSm.vue

```diff
@@ -249,7 +249,7 @@ $icon-width: 2.75rem;
 // Complete filing required styling
 .complete-filing {
   color: $gray6;
-  font-size: 0.85rem;
+  font-size: $px-14;
```

Comment:
This isn't an exact replacement, but the purpose of this font size is to be smaller than 15, which this is. (No one can tell the different between 0.85rem and 0.875rem.)

I also did this in a couple of other components where the font size was 0.8, 0.85 or 0.9.